stats/pages_proofed_graphs.php

```diff
@@ -11,29 +11,23 @@
 $tally_name = get_enumerated_param($_GET, 'tally_name', null, $valid_tally_names);
 
 $title = sprintf(_('Graphs for Pages Saved-as-Done in Round %s'), $tally_name);
-$charts = [
-    ["id" => "pages_daily_increments_curr_month", "config" => pages_daily($tally_name, "increments", "curr_month")],
-    ["id" => "pages_daily_cumulative_curr_month", "config" => pages_daily($tally_name, "cumulative", "curr_month")],
-    ["id" => "pages_daily_increments_prev_month", "config" => pages_daily($tally_name, "increments", "prev_month")],
-    ["id" => "pages_daily_increments_all_time", "config" => pages_daily($tally_name, "increments", "all_time")],
-    ["id" => "pages_daily_cumulative_all_time", "config" => pages_daily($tally_name, "cumulative", "all_time")],
-    ["id" => "total_pages_by_month_graph", "config" => total_pages_by_month_graph($tally_name)],
+$graphs = [
+    ["barChart", "pages_daily_increments_curr_month", pages_daily($tally_name, "increments", "curr_month")],
+    ["barChart", "pages_daily_cumulative_curr_month", pages_daily($tally_name, "cumulative", "curr_month")],
+    ["barChart", "pages_daily_increments_prev_month", pages_daily($tally_name, "increments", "prev_month")],
+    ["barChart", "pages_daily_increments_all_time", pages_daily($tally_name, "increments", "all_time")],
+    ["barChart", "pages_daily_cumulative_all_time", pages_daily($tally_name, "cumulative", "all_time")],
+    ["barChart", "total_pages_by_month_graph", total_pages_by_month_graph($tally_name)],
 ];
 
-$js_data = '$(function(){';
-foreach ($charts as $chart) {
-    $js_data .= 'barChart("' . $chart["id"] . '", ' . json_encode($chart["config"]) . ');';
-}
-$js_data .= '});';
-
 output_header($title, SHOW_STATSBAR, [
     "js_files" => get_graph_js_files(),
-    "js_data" => $js_data,
+    "js_data" => build_svg_graph_inits($graphs),
 ]);
 echo "<h1>$title</h1>";
 
 echo "<div style='max-width: 640px'>";
-foreach ($charts as $chart) {
-    echo "<div id='" . $chart["id"] . "' style='max-height: 400px'></div><hr>";
+foreach ($graphs as [$type, $id]) {
```

Comment:
Does this work? Elements of `$graphs` have 3 items in each array. But perhaps PHP discards extras? I honestly don't know.

Reply:
Whoops, I forgot to test that last night. I just did, and the pages proofed graphs are not showing at all (as compared with the main TEST sandbox, just to make sure it wasn't a data problem).

Reply:
It was due to my silly error with ?? usage. As for the original question, I don't see it listed in the list documentation (which [] in the LHS is a shorthand for)
https://www.php.net/manual/en/function.list.php
but I verified it does seem to just grab the first elements since I don't have any empty commas.

Reply:
I guess that means I could skip the $type since I'm not using it with a empty comma, but that feels strange to me :). Let me know if you prefer it.

Reply:
I prefer the way you have it!